script.js

```diff
@@ -1,5 +1,47 @@
+
 const playButton = document.querySelector('.play')
 const playMenu = document.querySelector('.gameButtons')
+const button1 = document.querySelector('.btn1')
+const button2 = document.querySelector('.btn2')
+const button3 = document.querySelector('.btn3')
```

Comment:
Is it most appropriate to use classes or IDs for your buttons?

Reply:
thank you :)